.circleci/config.yml

```diff
@@ -74,13 +69,58 @@ jobs:
       - checkout
       - python/install-packages:
           pkg-manager: pip
+      - run:
+          name: wake up serverless db
+          command: |
+            cd test/integration
+            dbt debug --target azuresql_sqlcred
       - run:
           name: Test adapter on Azure SQL against dbt-adapter-tests
           command: tox -e integration-azuresql
+  connection-azuresql:
+    docker: 
+      - image: dataders/pyodbc:1.4
+    executor: python/default 
+    steps:
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+      - run:
+          name: install gnupg
+          command: apt-get install gnupg -y
+      - azure-cli/install
+      - run: *prep=connect
+      - run:
+          name: cnxn -- Azure SQL - SQL CRED user+pass
+          command: |
+            cd test/integration
+            dbt compile --target azuresql_sqlcred
+      - azure-cli/login-with-user:
```

Comment:
To which Azure account is this login? Or is this something that works for testing only?

Reply:
i have a sandbox Azure tenant, that we're using just for CI purposes for all the dbt-msft repos. happy to share some of the creds if you'd like

Reply:
That is alright, I was curious how the integration test with Azure was setup.